pkg/daemon/ceph/client/command.go

```diff
@@ -54,23 +53,22 @@ func CephConfFilePath(configDir, clusterName string) string {
 }
 
 // FinalizeCephCommandArgs builds the command line to be called
-func FinalizeCephCommandArgs(command string, clusterInfo *ClusterInfo, args []string, configDir string) (string, []string) {
+func FinalizeCephCommandArgs(command string, clusterInfo *ClusterInfo, timeout time.Duration, args []string, configDir string) (string, []string) {
```

Comment:
We expect all of the commands to use the same timeout, right? If we expect all of the commands to use the same timeout from the operator setting, then it seems like we could simplify by not exposing a new param for this method. Where directly needed, this timeout could be used. For example, we could keep the `exec.CephCommandTimeout` as a var (instead of a const). The operator would set that var, then most of the code changes in this PR wouldn't be needed.